### PR TITLE
Add cabbage apps for 2022-01-21

### PR DESCRIPTION
**Pull request by JosephSalisbury**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Review by webwurst — APPROVED** [review]

File: src/content/changes/highlights/2022-01-21.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ owner:
 
 ## Apps
 
+- [kong-app](https://github.com/giantswarm/kong-app) [v2.5.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#250---2022-01-18) adds startup probes to allow for longer boot times in large clusters.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.7.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#270---2022-01-19) updates to controller image v1.1.1, among other improvements.
```
Comment (ubergesundheit) [inline src/content/changes/highlights/2022-01-21.md:13]: maybe include that previous versions are not compatible with k8s > 1.22

Comment (JosephSalisbury) [inline src/content/changes/highlights/2022-01-21.md:13]:
```suggestion
- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.7.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#270---2022-01-19) updates to controller image v1.1.1, among other improvements. Please note, previous versions are not compatible with Kubernetes versions greater than 1.22.
```

Comment (JosephSalisbury) [reply src/content/changes/highlights/2022-01-21.md:13]: @ubergesundheit how's that?

Comment (JosephSalisbury) [reply src/content/changes/highlights/2022-01-21.md:13]: i go with this

**Pull request merged**